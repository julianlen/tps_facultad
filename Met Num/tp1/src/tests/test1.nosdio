1 - 0.436466
4 - 0.478732
2 - 0.481496
5 - 0.406911
6 - 0.740618
3 - 0.830665
