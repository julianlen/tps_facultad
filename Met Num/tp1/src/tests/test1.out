0.410191
0.440844
0.383438
0.346985
0.628813
0.789729
